client/tests/integration/asset_propagation.rs

```diff
@@ -18,17 +18,14 @@ fn client_add_asset_quantity_to_existing_asset_should_increase_asset_amount_on_a
     wait_for_genesis_committed(&network.clients(), 0);
     let pipeline_time = Configuration::pipeline_time();
 
-    let create_domain = RegisterBox::new(IdentifiableBox::Domain(
-        Domain::new(DomainId::new("domain")?).into(),
-    ));
+    let create_domain = RegisterBox::new(Domain::new(DomainId::new("domain")?));
     let account_id = AccountId::new("account", "domain")?;
-    let create_account = RegisterBox::new(IdentifiableBox::NewAccount(
-        NewAccount::with_signatory(account_id.clone(), KeyPair::generate()?.public_key).into(),
+    let create_account = RegisterBox::new(Account::new(
+        account_id.clone(),
+        [KeyPair::generate()?.public_key],
```

Comment:
@arndey , this might be what was preventing you from adding new signatories in #1969.